src/main/java/com/facebook/openwifirrm/DeviceDataManager.java

```diff
@@ -539,4 +552,16 @@ public void updateDeviceApConfig(DeviceApConfigFunction fn) {
 		cachedDeviceConfigs.clear();
 		saveDeviceLayeredConfig();
 	}
+
+	/**
+	 * Return config for the given zone with all config layers applied. It is
+	 * the caller's responsibility to ensure the zone exists in the topology.
```

Comment:
I guess the assumption here is that a zone (i.e., venue in owprov) will be created while (or before) RRM scheduler is being set, is this the correct understanding? Thanks.

Reply:
Sort of, not exactly.
- `ProvMonitor` calls `scheduler.syncTriggers()` after it syncs topology from owprov, so this is handled.
- We also want to decouple the scheduler from owprov such that the scheduler works even if owprov is not deployed.